src/app/patient-dashboard/patient-search/patient-search.component.html

```diff
@@ -20,7 +19,7 @@
 				<div class="clear"></div>
 			</div>
 		</form>
-		<div *ngIf="totalPatients > 0" class="alert"><span class="total text-bold" (ngModel)="totalPatients">{{totalPatients}}</span>
+		<div *ngIf="hasConductedSearch===true && searchString.length > 0" class="alert"><span class="total text-bold" (ngModel)="totalPatients">{{totalPatients}}</span>
 			<span *ngIf="totalPatients <= 1"> patient found</span>
```

Comment:
Change this to to '0 results found' or something better. '0 patient found' sounds funny